src/index.ts

```diff
@@ -1,81 +1,128 @@
-}
+  /**
+   * Generates an output log with severity level INFO.
+   *
+   * @param {String} message  is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
+  public info (message: string, payload: Object): void {
+    logger.info(this._buildPayload(LogLevel.INFO, message, payload))
+  }
 
-/**
- * Logs a warn log.
- *
- * @param message is a JSON object that represents the body of the event.
- * @param event  is the name of request event (request_started, request_finished and etc)
- * @param messageEvent is the name of the system event (BookingFound, BookingNotFound and etc). The default value is 'request'.
- */
-// prettier-ignore
-function warn (message: unknown, event: string, messageEvent: string): void {
-  console.warn(createMessage(message, event, messageEvent, LogLevel.WARN))
-}
+  /**
+   * Generates an output log with severity level WARN.
+   *
+   * @param {String} message  is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
+  public warn (message: string, payload: Object): void {
+    logger.warn(this._buildPayload(LogLevel.WARN, message, payload))
+  }
+
+  /**
+   * Generates an output log with severity level ERROR.
+   *
+   * @param {String} message  is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
+  public error (message: string, payload: Object): void {
+    logger.error(this._buildPayload(LogLevel.ERROR, message, payload))
+  }
 
-/**
- * Logs an error log.
- *
- * @param message is a JSON object that represents the body of the event.
- * @param event  is the name of request event (request_started, request_finished and etc)
- * @param messageEvent is the name of the system event (BookingFound, BookingNotFound and etc). The default value is 'request'.
- */
-// prettier-ignore
-function error (message: unknown, event: string, messageEvent: string): void {
-  console.error(createMessage(message, event, messageEvent, LogLevel.ERROR))
+  /**
+   * Generates an output log with severity level TRACE.
+   *
+   * @param {String} message  is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
+  public trace (message: string, payload: Object): void {
+    logger.trace(this._buildPayload(LogLevel.TRACE, message, payload))
+  }
+
+  /**
+   * Creates the log structure.
+   *
+   * @param {String} severity the log level.
+   * @param {String} message  is the description of log(i.e. "Lorem ipsum dolor sit amet").
+   * @param {Object} payload is a JSON object that represents the content of the event.
+   */
+  private _buildPayload (
```

Comment:
[Suggestion]
Make the description and the name more consistent.

my preferred option
```suggestion
  /**
   * Creates the log entry.
   *
   * @param {String} severity the log level.
   * @param {String} message  is the description of log(i.e. "Lorem ipsum dolor sit amet").
   * @param {Object} payload is a JSON object that represents the content of the event.
   */
  private _buildLog (
```
**OR**

```suggestion
  /**
   * Creates the log structure.
   *
   * @param {String} severity the log level.
   * @param {String} message  is the description of log(i.e. "Lorem ipsum dolor sit amet").
   * @param {Object} payload is a JSON object that represents the content of the event.
   */
  private _buildPayload (
```
```suggestion
  /**
   * Creates the log payload.
   *
   * @param {String} severity the log level.
   * @param {String} message  is the description of log(i.e. "Lorem ipsum dolor sit amet").
   * @param {Object} payload is a JSON object that represents the content of the event.
   */
  private _buildPayload (
```